spec/dummy/app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+class User < ActiveRecord::Base
+end
```

Comment:
If this model needed for fixtures I think the name should be consistent, we can have a users fixture instead of `blah`